### DISCUSSION TITLE
用于webdav文件上传的temp缓存

### DISCUSSION THREAD

**Discussion by marcmk6** [AlistGo/alist#6213] [Q&A]
上传文件到webdav挂载的存储，似乎是会先把文件存到temp文件夹里再进行上传。这个上传过程的并发程度是如何的：同时上传全部位于temp的文件？还是有一个上限？

**Comment by pongfcnkl** [comment]
看你webdav客户端的了
webdav客户端上传的多了就全部都去temp了

**Reply by marcmk6** [reply]
原来是这样。那**理论上**有没有可能在alist这边限制某个webdav客户端存到temp的文件数量？我的使用情况下webdav客户端是专有闭源软件，所以无法控制其上传行为

**Reply by pongfcnkl** [reply]
用rclone中转？
